src/components/QuoteBox/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+interface QuoteBox {
+  readonly quote: string
+}
+
+const QuoteBox = ({ quote }: QuoteBox) => {
+  const { t } = useTranslation('about-us')
+
+  return (
+    <div className="quote-box">
+      <header className="quote-box__header">
+        <h3 className="quote-box__title">{t('quoteSectionHeader')}</h3>
+      </header>
+      <div className="quote-box__container quote-box__container--background-image">
+        <p className="quote-box__content">{quote}</p>
+      </div>
```

Comment:
`quote-box__container` inside `quote-box` just doesn't make sense when both of them are a block element.